x11-base/xorg-proto/xorgproto-2022.2.recipe

```diff
@@ -120,4 +119,13 @@ BUILD()
 INSTALL()
 {
 	make install
+
+	# licenses
+	mkdir -p $dataDir/licenses/xorgproto
+	cp COPYING-* $dataDir/licenses/xorgproto
+
+	# cleanup
+	rm -f $dataDir/licenses/xorgproto/COPYING-{applewmproto,windowswmproto}
+	rm -f $dataDir/pkgconfig/applewmproto.pc
+	rm -f $includeDir/X11/extensions/apple*
```

Comment:
These shouldn't be removed.

Reply:
I added this after checking some recipes over at Repology.
Will remove it, thanks for checking.

Reply:
Fixed.

Reply:
Which recipes over at Repology? Does Gentoo keep or remove them?

Reply:
Gentoo keeps them, Arch removes them, Fedora removes them ...  :) (so it's a bit of a mix)

https://git.alpinelinux.org/aports/tree/main/xorgproto/APKBUILD
https://gitlab.archlinux.org/archlinux/packaging/packages/xorgproto/-/blob/main/PKGBUILD
https://src.fedoraproject.org/rpms/xorg-x11-proto-devel/blob/rawhide/f/xorg-x11-proto-devel.spec
https://gitweb.gentoo.org/repo/gentoo.git/tree/x11-base/xorg-proto/xorg-proto-2022.2.ebuild

Reply:
Let's stick with what Gentoo does.